hbase-it/src/test/java/org/apache/hadoop/hbase/IntegrationTestBackupRestore.java

```diff
@@ -139,7 +139,7 @@ public void tearDown() throws IOException {
   public void setUpMonkey() throws Exception {
     Policy p = new PeriodicRandomActionPolicy(sleepTime,
       new RestartRandomRsExceptMetaAction(sleepTime));
-    this.monkey = new PolicyBasedChaosMonkey(util, p);
+    this.monkey = new PolicyBasedChaosMonkey(null, util, p);
```

Comment:
nit: If we just overload the constructor on PolicyBasedChaosMonkey, then we don't need this extra change just to pass a null reference.

Reply:
You are right, I'll fix this.